bc_ccv_rstudio_singularity/install-compute-dependencies.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Install easy repo dependencies
+yum-config-manager --add-repo https://turbovnc.org/pmwiki/uploads/Downloads/TurboVNC.repo
+yum install -y epel-release
+yum install -y nc python-websockify singularity turbovnc 
+
+# Install RStudio from RPM
```

Comment:
so rstudio doesn't need to be in the container already, just a particular version of R?

Reply:
no it does not need Rstudio, but i presume you are using a rocker container. So you would just pull that container into OSCAR  and this way you are using the same container both on your laptop and on RStudio